apisix/plugins/error-log-logger.lua

```diff
@@ -62,14 +63,18 @@ local log_level = {
     DEBUG  =    ngx.DEBUG
 }
 
+
 local config = {
 }
 
+
 local _M = {
     version = 0.1,
     priority = 1091,
     name = plugin_name,
     schema = schema,
+    metadata_schema = schema,
```

Comment:
that is wrong. they are different between `schema` and `metadata_schema`.

Reply:
I think the metadata_schema define the config used by the plug-in, Is there sth wrong?